staging/reset.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+cd windows
+start /min gpii-app.exe --reset
```

Comment:
Will need to change this for https://github.com/GPII/gpii-wix-installer/pull/42

Reply:
Yes, let's do it after #42 is merged